tooling/nargo_cli/tests/execution_success/brillig_schnorr/src/main.nr

```diff
@@ -1,7 +1,9 @@
 use dep::std;
 // Note: If main has any unsized types, then the verifier will never be able
 // to figure out the circuit instance
-unconstrained fn main(message: [u8; 10], message_field: Field, pub_key_x: Field, pub_key_y: Field, signature: [u8; 64]) {
+unconstrained fn main(
+    message: [u8; 10], message_field: Field, pub_key_x: Field, pub_key_y: Field, signature: [u8; 64]
+) {
```

Comment:
I think we should avoid this form of formatting where possible. I think we should have function signatures all on one line or go to the fully vertical form.

Reply:
@f01dab1e